source/2020-08-28-the-ember-times-issue-162.md

```diff
@@ -9,13 +9,24 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
   
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Update from the Accessibility Working Group 🙌,
 Syntax highlighting in markdown with ember-showdown-prism 🔦,
 Read the proposed `{{id}}` helper RFC 🆔,
 
 READMORE
 
 ---
 
+## [Update from the Accessibility Working Group 🙌](https://blog.emberjs.com/2020/08/25/accessibility-working-group-update.html)
+
+In March 2020, a group of Ember developers formed the Accessibility (A11y) Strike Team, now known as the **Ember A11y Working Group**. The group wanted to address the issues that were outlined in [RFC #595: Technical A11y Issues in New Ember Apps](https://github.com/emberjs/rfcs/issues/595).
+
+This week, [Melanie Sumner (@Melsumner)](https://github.com/MelSumner) published an [official blog post](https://blog.emberjs.com/2020/08/25/accessibility-working-group-update.html) to provide a status update. We encourage you to [read the blog post](https://blog.emberjs.com/2020/08/25/accessibility-working-group-update.html). Accessible apps are for everyone and the discussions, RFCs, and addons started by the A11y Strike Team may influence your app development.
```

Comment:
```suggestion
This week, [Melanie Sumner (@Melsumner)](https://github.com/MelSumner) published an [official blog post](https://blog.emberjs.com/2020/08/25/accessibility-working-group-update.html) to provide a status update. In addition to reading the [blog post](https://blog.emberjs.com/2020/08/25/accessibility-working-group-update.html), we encourage you to drop into [#dev-ember-a11y](https://discord.com/channels/480462759797063690/680503382036840496) on the [Ember Discord](https://discord.com/invite/emberjs). Accessible apps are for everyone and the discussions, RFCs, and addons started by the A11y Strike Team may influence your app development.
```

just a suggestion to make it less redundant, feel free to ignore 

i feel badly that i am mentioned in that post... i only participated at the beginning 😬

Reply:
Hmm, I think linking to Discord feels out of place. Since the blog post has the link, I think we can let a reader visit the post, then decide whether to visit Discord?